### PR TITLE
GH 7066 (code block formatting and standardization issues) - `prepare-mattermost-database.rst`

### PR DESCRIPTION
**Pull request by chessmadridista**
#### Summary

This pull request fixes the redundant background issue of the code blocks. Only one file named `prepare-mattermost-database.rst` has been fixed in this request.

#### Ticket Link

Related to #7066

### CODE REVIEW

**Comment by cwarnermm** [comment]
Thanks, @chessmadridista - Given the substantial number of merge conflicts in this docs PR, can you summarize the code-level changes applied to the code blocks, please? It looks like you changed defined code blocks, such as ``.. code-block:: sh`` to ``.. code-block:: none`` to fix the redundant background issue of the code blocks. Is that correct?

**Comment by chessmadridista** [comment]
> Thanks, @chessmadridista - Given the substantial number of merge conflicts in this docs PR, can you summarize the code-level changes applied to the code blocks, please? It looks like you changed defined code blocks, such as ``.. code-block:: sh`` to ``.. code-block:: none`` to fix the redundant background issue of the code blocks. Is that correct?

Oh, really sorry about that. I realised right now that I pulled the changes in my remote fork but forgot to pull them in my local repo. I'll make the changes after pulling in the updates. 

As for the change to be done, it is a simple indentation issue. I'll do the changes and update you accordingly. 

I'll create a new PR and will let you know.
